com.github.dictation_toolbox.Caster.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: io.github.dictation_toolbox.Caster
```

Comment:
the manifest filename doesn't match this.

Reply:
fixed this